core/outbox.go

```diff
@@ -35,8 +36,8 @@ type Outbox struct {
 }
 
 type outboxChainProvider interface {
```

Comment:
This is identical to the `inboxChainProvider` interface. Seeing as they both live in the same package, is there any reason to separate them like this?